doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -104,40 +107,80 @@ execution and result reporting along with required artifacts and environments in
 place.
 
 #### 5.3.1 Test Case Model
+As there are more number of VNF at different levels of networking such as access,
+transport and core level as well as OSI level L0-L7. Every network functions
+provides set of pre-defined features and functionalities. so its important to

```

Comment:
Sentence issue, "so its important to model test cases for every functionality to identify it uniquely and use it as
part of test flow design." Is this about modeling functionality and building test cases -- not sure about modeling test cases?

Capitalise "s" in "so"

Reply:
sure. thanks

Reply:
@pgoyal01 - it's about establishing test cases for types of VNFs requiring certain performance, services, or capabilities, from the NFVI (infrastructure).  The objective would be to categorize VNFs into B/N/C (Basic, Network, or Compute) groups, or families, then based on these families, establish a template, or Model, per family, along with the respective test cases to verify VNF platform meets stated requirements.  The Model referenced is simply a way to represent the family we're testing, and the parameters needed for testing that family.

@pgoyal01 - can we close/resolve?

Reply:
@mf4716 Can you clarify (as in your response) so that it is clear

Reply:
@pgoyal01 - please re-read section 5.3.1.  Bottom line, we are attempting to normalize the collection of inputs needed to build a Certification and Verification framework for VNFs based on VNF family, with unique characteristics if they fall into a Basic, Network, or Compute, category based on their design, and needs from the NFVI.

Excerpts from 5.3.1 to hopefully help clarify the comments in the dialogue:

"As there are more number of VNF at different levels of networking such as access, transport and core level.. its important to model [i.e. create a template for capturing normalized test case requirements based on VNF family] test cases [to properly test VNFs and their respective families].. As part of modeling its very important to capture the following details:

Test case Name
Test case description
Virtual Network function Name
Network function Feature/functionality name
...etc
...while implementing the test cases, this model would act as specification and as it captures the input and output, it would help while designing the test flow which will help to execute set of test cases in pre-defined flow."

Again, the goal of this section are to normalize the collection of data to help define and design test cases/scenarios for VNF certifications.  i.e. "a Framework", which we're accomplishing in part, in a normalized table format.

Reply:
@mf4716 OK - Thanks